examples/collapse-groups.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>OpenLayers 3 - LayerSwitcher</title>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no, width=device-width">
+    <link rel="stylesheet" href="https://openlayers.org/en/v5.1.3/css/ol.css" />
+    <link rel="stylesheet" href="../src/ol-layerswitcher.css" />
+    <link rel="stylesheet" href="layerswitcher.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
```

Comment:
I'd prefer it if we could avoid adding a dependency on fontawesome; the arrows could be either pure CSS (along the lines of https://css-tricks.com/the-shapes-of-css/) or base64 encoded images in the stylesheet.